.github/workflows/template_testio_trigger_test.yml

```diff
@@ -0,0 +1,33 @@
+name: TestIO - Trigger Test
+
+on:
+  workflow_call:
+    inputs:
+      testio-slug:
+        required: false
+        type: string
+        default: 'staffbase'
+      testio-product-id:
+        required: true
+        type: string
+    secrets:
+      github-token:
+        required: true
+      testio-token:
+        required: true
+
+jobs:
+  testio-trigger-test:
+
+    name: TestIO - Trigger Test
+    runs-on: ubuntu-22.04
+    if: startsWith(github.event.comment.body, '@bot-testio exploratory-test')     # this is the prefix all subsequent comments must start with
+
+    steps:
+      - name: Request required input and trigger test on TestIO
+        uses: Staffbase/testio-trigger-test-github-action@main
```

Comment:
Is the plan to open source the `Staffbase/testio-trigger-test-github-action` action in future? Otherwise I would argue against having a public reusable workflow with a private action.

Reply:
It will be public in the future. 🙌

Reply:
Yes, once [this Legal](https://mitarbeiterapp.atlassian.net/servicedesk/customer/portal/8/LP-870) request got approved.